database/model.js

```diff
@@ -1 +1,25 @@
 import db from "./connection";
+
+async function addArt(mood, alt, caption, pub, prompt, img) {
```

Comment:
really super work on the async and await functionality for all the moods and description, image uploads 🧪 🖌️